docs/guides/component-testing/getting-started.mdx

```diff
@@ -0,0 +1,622 @@
+
+```bash
+npm install cypress -D
+```
+
+### Open Cypress:
+
+Once Cypress has been installed you can launch Cypress using the following command:
+
+```bash
+npx cypress open
+```
+
+Next, the Cypress Launchpad will guide you through configuring your project.
+
+## Configuring Component Testing
+
+### Select Testing Type
+
+Whenever you run Cypress for the first time, the app will prompt you to set up
+either E2E Testing or Component Testing. Click on "Component Testing" to start
+the configuration wizard.
+
+<DocsImage
+  src="/img/guides/component-testing/select-test-type.jpg"
+  caption="Choose Component Testing"
+/>
+
+### Project Setup
+
+The Project Setup screen will automatically detect your framework and bundler.
+Click "Next Step" to continue.
+
+<DocsImage
+  src="/img/guides/component-testing/project-setup-react.jpg"
+  caption="React and Vite are automatically detected"
+/>
+
+### Installing Dependencies
+
+The next screen checks that all the required dependencies are installed. All the
+items should have green checkboxes on them, indicating everything is good, so
+click "Continue".
+
+<DocsImage
+  src="/img/guides/component-testing/dependency-detection-react.jpg"
+  caption="All necessary dependencies are installed"
+/>
+
+### Config Files
+
+Next, Cypress generates all the necessary configuration files and gives you a
+list of all the changes it made to your project. Click "Continue".
+
+<DocsImage
+  src="/img/guides/component-testing/scaffolded-files.jpg"
+  caption="The Cypress launchpad will scaffold all of these files for you"
+/>
+
+### Choose A Browser
+
+After setting up component testing, you will be at the browser selection screen.
+
+Pick the browser of your choice and click the "Start Component Testing" button
+to open the Cypress test runner.
+
+<DocsImage
+  src="/img/guides/component-testing/select-browser.jpg"
+  caption="Choose your browser"
+/>
+
+## Writing Your First Test
+
+At this point, your project is configured to use Cypress Component Testing. However, when the test runner appears, it won't find any specs because we haven't created
+any yet. Now we need write our first Component Test.
+
+<DocsImage src="/img/guides/component-testing/create-your-first-spec-cfc.jpg" />
+
+### Frameworks
+
+Cypress currently has official mounting libraries for
+[React](/guides/component-testing/react/overview),
+[Angular](/guides/component-testing/angular/overview),
+[Vue](/guides/component-testing/vue/overview), and
+[Svelte](/guides/component-testing/svelte/overview).
+In addition to our official framework support we also have many community plugins such as
+[Qwik](https://github.com/qwikifiers/cypress-qwik) and [Lit](https://github.com/redfox-mx/cypress-lit).
+
+### Your First Component Test
+
+Though every framework is different and have their own unique nuances, in general, writing tests are very similar.
+Let's look at how a basic test is written for a `StepperComponent`.
+
+:::note
```

Comment:
This white-on-white note ends up looking like an aside to me, I skipped it initially. I wonder if it could be a regular paragraph, or some more prominent kind of note.

![Screenshot 2023-07-26 at 10 48 45 AM](https://github.com/cypress-io/cypress-documentation/assets/8340719/e67bacbc-3aa3-43eb-857f-362c52cba742)